eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -217,9 +217,11 @@ services:
     eZ\Publish\Core\Pagination\Pagerfanta\AdapterFactory\SearchHitAdapterFactoryInterface:
         alias: eZ\Publish\Core\Pagination\Pagerfanta\AdapterFactory\SearchHitAdapterFactory
 
-    ezpublish.fields_groups.list:
-        class: eZ\Publish\Core\Helper\FieldsGroups\ArrayTranslatorFieldsGroupsList
-        factory: ["@ezpublish.fields_groups.list.repository_settings_factory", "build"]
+    ezpublish.fields_groups.list: '@eZ\Publish\Core\Helper\FieldsGroups\ArrayTranslatorFieldsGroupsList'
+
+    eZ\Publish\Core\Helper\FieldsGroups\ArrayTranslatorFieldsGroupsList:
+        lazy: eZ\Publish\Core\Helper\FieldsGroups\FieldsGroupsList
```

Comment:
Is this a right syntax for lazy?

Reply:
Yes: https://github.com/symfony/symfony/pull/27697

Reply:
I don't see any documentation about it at all, but I've looked through Symfony code and it does make sense.

When `lazy` option contains a string, it is tagged for proxy manager to consume and adjusts the resulting class.